src/com/mani/lang/Modules/std/std.java

```diff
@@ -117,6 +118,7 @@ public int arity() {
             public Object call(Interpreter interpreter, List<Object> arguments) {
                 interpreter.wipeMemory();
                 Std.loaded.clear();
+                Mani.hadRuntimeError = false;
```

Comment:
Made it so hat wiping memory also wipes the runtime error flag. This is to make sure the tests after those which are expected to error do not cause following tests to error out.

Reply:
Good thinking. I would have overlooked this.